classes/game.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'item'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at
+
+  def initialize(multiplayer, last_played_at, publish_date)
```

Comment:
Kindly add a default value to variables, as a way to specify their type.
```suggestion
  def initialize(multiplayer: false, last_played_at, publish_date)
```

Reply:
totally agree 💯